---
title: Short Paper
subtitle: Subtitle
author:
  - name: Tran Quoc Hoang
    email: tranquochoang@protonmail.com
    affiliations: 
        - id: international-uni
          name: International University - VNU-HCM
          department: School of Biotechnology
          address: Quarter 33, Linh Xuan Ward
          city: Ho Chi Minh City
          state: ~
          postal-code: 71309
          country: Vietnam
    attributes:
        corresponding: true
    note: "Student ID: BTBTWE24036"

  - name: Nguyen Hoang Thanh Ngan
    email: "BTBTIU24067@student.hcmiu.edu.vn"
    affiliations:
        - id: international-uni
    note: ~

  - name: Pham Gia Han
    email: "BTBTWE24003@student.hcmiu.edu.vn"
    affiliations:
        - id: international-uni
    note: ~

  - name: Pham Ngo Phuong Thao
    email: "BTBTIU24099@student.hcmiu.edu.vn"
    affiliations:
        - id: international-uni
    note: ~
    
  - name: Nguyen Hoang Tuong Vy
    email: "BTBTIU24133@student.hcmiu.edu.vn"
    affiliations:
        - id: international-uni
    note: ~

  - name: Nguyen Le Ngoc Vy
    email: "BTBTIU24131@student.hcmiu.edu.vn"
    affiliations:
        - id: international-uni
    note: ~

abstract: |
  This is the abstract. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum augue turpis, dictum non malesuada a, volutpat eget velit. Nam placerat turpis purus, eu tristique ex tincidunt et. Mauris sed augue eget turpis ultrices tincidunt. Sed et mi in leo porta egestas. Aliquam non laoreet velit. Nunc quis ex vitae eros aliquet auctor nec ac libero. Duis laoreet sapien eu mi luctus, in bibendum leo molestie. Sed hendrerit diam diam, ac dapibus nisl volutpat vitae. Aliquam bibendum varius libero, eu efficitur justo rutrum at. Sed at tempus elit.
keywords: 
  - keyword1
  - keyword2
date: last-modified
bibliography: bibliography.bib
format:
  elsevier-pdf:
    includes:
      in_header: preamble.tex
    keep-tex: true
    # cite-method: citeproc # Default is `natbib`
    journal:
      name: ~
      formatting: preprint
      model: 5p # Don't set a model with preprint. Specify one or two columns only when using 3p.
      # layout: twocolumn
      # cite-style: authoryear
      cite-style: number
      # cite-style: super # Not supported by `citeproc`
    # csl: elsevier-vancouver.csl
---

```{=html}
<!--
comment template, so I don't forget how to do it lol
-->
```

```{r renv-dependencies, include=FALSE}
# This chunk makes the core R rendering packages visible to renv::snapshot()
# The 'knitr' and 'rmarkdown' packages pull in most of the other dependencies.
library(knitr)
library(rmarkdown)
library(tinytex) 
library(yaml)
library(ragg)
```

\begin{center}
\textbf{Submission Note:} Paper prepared for the Genetics (BT313IU) course, International University - VNU-HCM, under the instruction of MSc. Hang T. Tong.
\end{center}

# Introduction

Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa. Vestibulum lacinia arcu eget nulla. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.

## Context and Background

Curabitur sodales ligula in libero. Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. Maecenas mattis. Sed convallis tristique sem. Proin ut ligula non magna fermentum. In a enim. Suspendisse sagittis.

## Scope and Thesis

Nunc a ullamcorper elit. Nulla facilisi. Aenean placerat ac neque vitae dictum. Vestibulum ac ante in sem tincidunt cursus. Sed at efficitur quam. Vivamus sed dictum magna. Vivamus laoreet, urna eget convallis malesuada, odio urna euismod risus, in euismod nibh odio in velit.

# Materials & Methods

Phasellus ultrices nulla quis nibh. Quisque a lectus. Donec consectetuer ligula vulputate sem tristique cursus. Nam nulla quam, gravida non, commodo a, sodales sit amet, nisi. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vestibulum a turpis vel dolor mattis tincidunt vel ac tortor.

## Literature Search Strategy

Donec ut dolor in massa auctor egestas. Nam eu massa. Sed convallis tristique sem. Proin ut ligula non magna fermentum. In a enim. Suspendisse sagittis. Fusce nec tellus sed augue semper porta. Mauris massa.

## Data Extraction and Synthesis

Sed hendrerit diam diam, ac dapibus nisl volutpat vitae. Aliquam bibendum varius libero, eu efficitur justo rutrum at. Sed at tempus elit. Nullam accumsan ipsum sit amet dui. Integer id odio.

# Discussion

Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Vivamus in erat ut urna cursus vestibulum. Morbi in dui quis est pulvinar ullamcorper. Vivamus sed dictum magna.

## Genetics in Etiology and Diagnosis

Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.

## Application in Therapy

PGx testing directly translates genetic data into evidence-based clinical decisions, which fundamentally alters the traditional trial-and-error approach to antidepressant therapy. By utilizing genetic data, clinicians can optimize treatment efficacy and safety through informed drug selection and precise dosage adjustments.

### Guiding Drug Selection

The primary application of PGx in antidepressant therapy is the initial selection of a medication that matches the patient's metabolic profile. This process aims to maximize efficacy while minimizing the risk of adverse drug reactions (ADRs). The Clinical Pharmacogenetics Implementation Consortium (CPIC) and the Dutch Pharmacogenetics Working Group (DPWG) provide guidelines for specific gene-drug pairs, particularly involving *CYP2D6* and *CYP2C19* [@bousmanClinicalPharmacogeneticsImplementation2023; @bousmanReviewConsensusPharmacogenomic2021].

For example, *CYP2D6* Ultrarapid Metabolizers (UMs) possess multiple active copies of the gene, resulting in significantly elevated enzyme activity. Table \ref{tab:tbl-cyp2d6-phenotype} and Table \ref{tab:tbl-cyp2c19-phenotype} show the assignment of likely phenotypes based on the diplotypes of the *CYP2D6* and *CYP2C19* genes, respectively. If these patients are prescribed Tricyclic Antidepressants (TCAs) such as amitriptyline or nortriptyline, the drug is metabolized and cleared from the body too quickly to achieve therapeutic blood concentrations [@AmitriptylineResponseNIH]. As such, standard treatments often have increased risk of failure. In those cases, guidelines recommend selecting an alternative antidepressant that is not primarily metabolized by *CYP2D6* to ensure the patient receives a therapeutic benefit [@deanAmitriptylineTherapyCYP2D62012].

\begin{table*}[!t]
  \begin{center}
    \caption{Assignment of predicted \textit{CYP2D6} phenotypes based on diplotypes}
    \label{tab:tbl-cyp2d6-phenotype}
    \begin{tabular}{p{3.8cm}|p{2.0cm}|p{5.8cm}|p{4.0cm}}
      \textbf{Phenotype} & \textbf{Activity score range} & \textbf{Genotypes} & \textbf{Examples of \textit{CYP2D6} diplotypes} \\
      \hline
      \textit{CYP2D6} Ultrarapid metabolizer ($\approx1–20\%$ of patients) 
      & $>2.25$ 
      & An individual carrying duplications of functional alleles 
      & $(*1/*1) \times N$, $(*1/*2) \times N$, $(*2/*2) \times N$ \\
      \textit{CYP2D6} Normal metabolizer ($\approx72–88\%$ of patients) 
      & $1.25\le x \le 2.25$
      & An individual carrying two normal function alleles or two decreased function alleles or one normal and no function allele or one normal function and decreased function allele or combinations of duplicated alleles that result in an activity score of $1.0–2.0$. 
      & $*1/*1$, $*1/*2$, $*2/*2$, $*1/*9$, $*1/*41$, $*41/*41$, $*1/*5$, $*1/*4$ \\
      \textit{CYP2D6} Intermediate metabolizer ($\approx1–13\%$ of patients) 
      & $0<x<1.25$
      & An individual carrying one decreased function and one no function allele
      & $*4/*41$, $*5/*9$, $*4/*10$ \\
      \textit{CYP2D6} Poor metabolizer ($\approx1–10\%$ of patients)
      & $0$
      & An individual carrying only no function alleles
      & $*4/*4$, $(*4/*4)\times N$, $*3/*4$, $*5/*5$, $*5/*6$ \\
      \textit{CYP2D6} Indeterminate
      & $0$
      & An individual carrying one or two uncertain and/or unknown function alleles
      & $*1/*22$, $*1/*25$, $*22/*25$
    \end{tabular}
    \vspace{1ex} % optional, a bit of vertical spacing
   {\footnotesize % or small font, depending on journal style
    \itshape \newline Note. Adapted from \cite{bousmanClinicalPharmacogeneticsImplementation2023} and \cite{hicksClinicalPharmacogeneticsImplementation2017}.}
  \end{center}
\end{table*}


\begin{table*}[!t]
  \begin{center}
    \caption{Assignment of predicted \textit{CYP2C19} phenotypes based on diplotypes}
    \label{tab:tbl-cyp2c19-phenotype}
    \begin{tabular}{p{3.8cm}|p{2.0cm}|p{5.8cm}|p{4.0cm}}
      \textbf{Phenotype} & \textbf{Activity score range} & \textbf{Genotypes} & \textbf{Examples of \textit{CYP2C19} diplotypes} \\
      \hline
      \textit{CYP2C19} Ultrarapid metabolizer
      & n/a 
      & An individual carrying two increased function alleles
      & $*17/*17$ \\
      \textit{CYP2C19} Rapid metabolizer
      & n/a
      & An individual carrying one normal function allele and one increased function allele
      & $*1/*17$ \\
      \textit{CYP2C19} Normal metabolizer
      & n/a
      & An individual carrying two normal function alleles
      & $*1/*1$ \\
      \textit{CYP2C19} Likely intermediate metabolizer
      & n/a
      & An individual carrying one normal function allele and one decreased function allele or one increased function allele and one decreased function allele or two decreased function alleles
      & $*1/*9$, $*9/*17$, $*9/*9$ \\
      \textit{CYP2C19} Intermediate metabolizer
      & n/a
      & An individual carrying one normal function allele and one no function allele or one no function allele and one increased function allele
      & $*1/*2$, $*1/*3$, $*2/*17$ \\
      \textit{CYP2C19} Likely poor metabolizer
      & n/a
      & An individual carrying one decreased function allele and one no function allele
      & $*2/*9$, $*3/*9$ \\
      \textit{CYP2C19} Poor metabolizer
      & n/a
      & An individual carrying two no function alleles
      & $*2/*2$, $*2/*3$, $*3/*3$ \\
      \textit{CYP2C19} Indeterminate
      & n/a
      & An individual carrying one or two uncertain function alleles
      & $*1/*12$, $*2/*12$, $*12/*14$ \\
    \end{tabular}
    \vspace{1ex} % optional, a bit of vertical spacing
   {\footnotesize % or small font, depending on journal style
    \itshape \newline Note. Adapted from \cite{bousmanClinicalPharmacogeneticsImplementation2023} and \cite{hicksClinicalPharmacogeneticsImplementation2017}.}
  \end{center}
\end{table*}

### Dosage Adjustment

### Treatment of Resistant Cases

## Clinical Challenges and Future Directions

Aliquam non laoreet velit. Nunc quis ex vitae eros aliquet auctor nec ac libero. Duis laoreet sapien eu mi luctus, in bibendum leo molestie. Nullam dictum felis eu pede mollis pretium.

# Abbreviations

| Abbreviation | Full Term              |
|--------------|------------------------|
| PGx          | Pharmacogenomic        |
| DSA          | Dolor sit amet         |
| TIU          | Tempor incididunt ut   |
| MV           | Minim veniam           |
| DIR          | Dolor in reprehenderit |

# Declaration of Competing Interest

The authors declare no competing interests.

# Data/Code Availability

The Quarto source code for this paper is available at \url{https://github.com/ht2905/pgx_antidepressant_response_article}.

# References
